custom_components/samsungtv_tizen/media_player.py

```diff
@@ -342,26 +393,40 @@ def _get_source(self):
                 self._source = self._running_app
         else:
             self._source = None
+            self._last_source_time = None
+            
         return self._source
 
     @util.Throttle(MIN_TIME_BETWEEN_SCANS, MIN_TIME_BETWEEN_FORCED_SCANS)
     def update(self):
         """Update state of device."""
-        if self._update_method == "smartthings" and self._api_key and self._device_id:
+        
+        #call_time = datetime.now()
```

Comment:
Please remove all unnecessary comments that include old / unused code.

Reply:
Done